kotlin_labs/kotlin_labs.graphql

```diff
@@ -36,7 +36,7 @@ directive @typePolicy(
     Since: 3.4.1
     """
     connectionFields: String! = ""
```

Comment:
Can you add more documentation here? Maybe a link to the relay spec and a small note about the `nodes` handling?

Reply:
👍  for the Relay spec

For `nodes` - didn't realize earlier but I think it doesn't make too much sense to mention it here, because its handling happens in `ConnectionFieldMerger`.

Reply:
Also, we should probably mark it 'experimental' as `paginationArgs`, right?

Reply:
Can you explicit this a bit more then. If I'm using `connectionFields`, looks like I need to use `ConnectionFieldMerger` at runtime? Do I need to do something else?

Reply:
> Also, we should probably mark it 'experimental' as paginationArgs, right?

Yes ✅ . We could use `@requiresOptIn` but let's keep that for another day

Reply:
> Can you explicit this a bit more then. If I'm using connectionFields, looks like I need to use ConnectionFieldMerger at runtime?

BTW, Another argument to separate those directives from `@typePolicy`. `@typePolicy` is probably going to be shared accross clients while `ConnectionFieldMerger` is probably Kotlin specific. But problem for another day.

Reply:
Also maybe it's a bit more straightforward to have 

```@graphql
extend type UserConnection @connection
```

than

```@graphql
extend type Query @typePolicy(connectionFields: "users")
```

Reply:
Good point 👍